### PR TITLE
Add 2.11 release post

### PR DESCRIPTION
**Pull request by joshmoore**
cc: @MSanKeys963 

fix: https://github.com/zarr-developers/zarr-python/issues/901

### CODE REVIEW

**Review by jni — APPROVED** [review]

**Comment by jni** [comment]
Slight note, possibly not important, but I think it would make more sense re sparse writing to say it was added by Davis and me, rather than me and Davis. He did the hard work on the implementation, I changed a flag from True to False 😂.

**Comment by MSanKeys963** [comment]
Sure @jni. 👍🏻

**Review by joshmoore — COMMENTED** [review]
Few points (I pushed commits for them)

File: _posts/2022-02-07-zarr-2.11-release.markdown [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+- [Gregory Lee](https://github.com/grlee77) 
+- [Ryan Abernathy](https://github.com/rabernat) 
+- [Matthias Bussonnier](https://github.com/Carreau) 
+- [Oren Watson](https://github.com/orenwatson) 
+- [Joe Hamman](https://github.com/jhamman) 
+- [Dimitri Papadopoulos Orfanos](https://github.com/DimitriPapadopoulos) 
+- [Ray Bell](https://github.com/raybellwaves) 
+- [John Kirkham](https://github.com/jakirkham) 
+- [Mads R. B. Kristensen](https://github.com/madsbk) 
+- [Josh Moore](https://github.com/joshmoore).
```
[the remark this answers is not in the document]
Comment (MSanKeys963) [reply _posts/2022-02-07-zarr-2.11-release.markdown:96]: Sure.

File: _posts/2022-02-07-zarr-2.11-release.markdown [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+If you find the above features useful and end up using them, please mention [@zarr_dev](https://twitter.com/zarr_dev) on Twitter and tweet using #zarr or #zarrdata and we'll make sure to get it featured! ✌🏻
```
[the remark this answers is not in the document]
Comment (MSanKeys963) [reply _posts/2022-02-07-zarr-2.11-release.markdown]: #ZarrData it is.

File: _posts/2022-02-07-zarr-2.11-release.markdown [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+categories: blog
+permalink: /release-2-11/
+---
+
+Version 2.11 of the [Python Zarr package](https://zarr.readthedocs.io/en/stable/) has just been released. 🎉 
+
+This blog post aims to provide an overview of new features in this release, especially a new parameter that may impact the performance of writing arrays.
+
+
+>### Empty chunks will no longer be written by default
```
[the remark this answers is not in the document]
Comment (MSanKeys963) [reply _posts/2022-02-07-zarr-2.11-release.markdown]: Sure.

**Comment by MSanKeys963** [comment]
All changes are done. Merging this PR now.

**Pull request merged**